creator/ingest_requests/mutations.py

```diff
@@ -37,12 +43,49 @@ def mutate(self, info, input):
         if not user.has_perm("ingest_requests.add_ingestrequest"):
             raise GraphQLError("Not allowed")
 
-        ingest_request = IngestRequest()
+        # Convert relay ids to Version objects
+        version_ids = input.get("versions", [])
+        versions = []
+        for vid in version_ids:
+            model, vid = from_global_id(vid)
+            try:
+                versions.append(Version.objects.get(kf_id=vid))
+            except Version.DoesNotExist:
+                raise GraphQLError(f"The Version {vid} does not exist")
+
+        # Create ingest request for a set of file versions
+        try:
+            with transaction.atomic():
+                # Create initial ingest request
+                ingest_request = IngestRequest()
+                ingest_request.save()
+                # Add versions to ingest request
+                ingest_request.versions.set(versions)
+                # Validate ingest request
+                ingest_request.full_clean()
+                ingest_request.save()
+        except ValidationError as e:
+            raise GraphQLError(f"Ingest request validation error: {str(e)}")
+        except IntegrityError as e:
+            if "duplicate key value" in str(e):
+                raise GraphQLError(
```

Comment:
Why can you only run an ingest once for a unique set of versions?
If my first ingest fails, shouldn't I be able to retry it?
Or if some later ingest overwrites this one, shouldn't I be able to run it again?

Reply:
I think it depends on how you think of an IngestRequest. I went back and forth on the concept of it but right now I'm thinking of itas a set of input parameters for an ingest job. Currently those parameters just consist of the set of files that will be ingested in the job. You should be able to execute an ingest job for a set of input parameters as 
many times as you want, but I don't think it makes sense to have multiple ingest jobs running at the same time for the same set of input parameters. I think that would be equivalent to running `kidsfirst ingest /path/to/ingest_package` on your machine in two different processes.